README.md

```diff
@@ -9,6 +9,21 @@ Video PreTraining (VPT): Learning to Act by Watching Unlabeled Online Videos
   :space_invader: [MineRL Environment](https://github.com/minerllabs/minerl) (note version 1.0+ required) \
   :checkered_flag: [MineRL BASALT Competition](https://www.aicrowd.com/challenges/neurips-2022-minerl-basalt-competition)
 
+
+## Running models
+
+Install requirements and [MineRL v1.0](https://github.com/minerllabs/minerl/tree/v1.0.0), and download models below.
```

Comment:
Can we make this a bit simpler to copy paste? e.g. 
`pip install -r requirements.txt`
`git clone minerl@branch`
`pip install -e ./minerl`
Or do you think that is too much?